docs/custom_domain_name_setup/README.md

```diff
@@ -0,0 +1,103 @@
+# Setup Custom Domain Name for QnABot Designer and Client
+This readme provides information on how to setup a custom domain name and configure AWS QnABot solution to use the custom domain name for the Designer, and Client user interfaces. 
+The setup and configuration involve the following steps:
+
+## Step 1: Setup custom domain name for API Gateway
+Use the AWS Account and AWS Region where you have deployed the AWS QnABot solution for the below steps. 
+Follow the steps outlined in the below link to setup:
+https://docs.aws.amazon.com/apigateway/latest/developerguide/how-to-custom-domains.html 
+
+-	Registering a domain name
+-	Creating DNS records
+-	Creating a SSL certificate for the custom domain name
+-	Creating a custom domain in API Gateway
+
+`Note: Disable the default API gateway endpoint since the custom domain name will be used.`
+
+
+## Step 2: Custom domain API Mapping setup in API Gateway
+When mapping the API to the custom domain in API Gateway for the QnABot deployment, use the below settings: 
+
+Create two mappings:
+
+##### Mapping 1: 
+- `API` = select the QnABot deployment you would like to use. The QnABot API will be named with the same name as the CloudFormation Stack name you used when you deployed the AWS QnABot solution
+- `Stage` = `prod`	- this is the default stage created for the QnABot deployment
+
+##### Mapping 2: 
+- `API` = select the QnABot deployment you would like to use. The QnABot API will be named with the same name as the CloudFormation Stack name you used when you deployed the AWS QnABot solution
+- `Stage` = `prod`	- this is the default stage created for the QnABot deployment
+- `Path` = `prod`	- for routing requests
```

Comment:
`Mapping 2` is exactly the same as `Mapping 1`, except the `Path` part?

Reply:
Thats correct. There are some API requests that require the path and some dont - being the case 2 mappings are used. 

API Gateway routes requests to the API mapping that has the longest matching path. API Gateway considers only the paths configured for API mappings, and not API routes, to select the API to invoke. If no path matches the request, API Gateway sends the request to the API that you've mapped to the empty path (none).